src/integration-test/java/apptest/IntegrationTest.java

```diff
@@ -101,57 +107,65 @@ void test01_standardPostBatchJob() throws JsonProcessingException, ClassNotFound
 			assertThat(archiveHistory.getBatchHistory()).isEqualTo(postBatchHistory);
 			assertThat(archiveHistory.getArchiveStatus()).isEqualTo(COMPLETED);
 		});
-		assertThat(archiveHistoryRepository.getArchiveHistoriesByBatchHistoryId(postBatchHistory.getId())).isEqualTo(getArchiveHistoryList);
+
+		getArchiveHistoryList.forEach(archiveHistory -> {
+			archiveHistory.setMunicipalityId(MUNICIPALITY_ID);
+			archiveHistory.getBatchHistory().setMunicipalityId(MUNICIPALITY_ID);
+		});
+		assertThat(archiveHistoryRepository.getArchiveHistoriesByBatchHistoryIdAndMunicipalityId(postBatchHistory.getId(), MUNICIPALITY_ID)).isEqualTo(getArchiveHistoryList);
 	}
 
 	// POST batch and then GET batch history and archive histories - verify that the correct is returned
 	@Test
 	void test02_standardPostBatchJob_zeroArchivedDocs() throws JsonProcessingException, ClassNotFoundException {
-		var batchJob = BatchJob.builder()
+		final var batchJob = BatchJob.builder()
 			.withStart(LocalDate.parse("2021-01-01"))
 			.withEnd(LocalDate.parse("2021-01-01"))
 			.build();
 
 		// POST batchJob
-		var postBatchHistory = postBatchJob(batchJob);
+		final var postBatchHistory = postBatchJob(batchJob);
 
 		// GET archiveHistory
 		setupCall()
 			.withHttpMethod(GET)
-			.withServicePath("/archived/attachments?batchHistoryId=" + postBatchHistory.getId())
-			.withExpectedResponseStatus(NOT_FOUND)
+			.withServicePath(ARCHIVED_PATH + "?batchHistoryId=" + postBatchHistory.getId())
+			.withExpectedResponseStatus(OK)
 			.withExpectedResponse("response.json")
-			.sendRequestAndVerifyResponse();
+			.sendRequestAndVerifyResponse()
+			.verifyAllStubs();
```

Comment:
I'm not sure, but isn't the stubs verified in the sendRequestAndVerifyResponse-method
```
	public AbstractAppTest sendRequestAndVerifyResponse() {
		return sendRequest().verifyStubs();
	}
```
?